js/docEnums.js

```diff
@@ -140,6 +140,11 @@
  * @enum {'center'|'end'|'start'|'stretch'}
  */
 
+/**
+ * @typedef {string} Enums.ButtonGroupSelectionMode
+ * @enum {'multiple'|'single'}
```

Comment:
Can we unify it with [this](https://github.com/DevExpress/DevExtreme/blob/18_2/js/docEnums.js#L170) and [this](https://github.com/DevExpress/DevExtreme/blob/18_2/js/docEnums.js#L510)?

Reply:
I not understand:)

Reply:
> Can we make one SelectionMode enum to use it in the Button group and use it to replace ChartElementSelectionMode and NavSelectionMode

It's a breaking change for MVC controls. If we want to do this, we should deprecate `ChartElementSelectionMode` and `NavSelectionMode` to keep backward compatibility.